crates/pallets/starknet/src/tests/declare_tx.rs

```diff
@@ -102,6 +105,7 @@ fn given_contract_declare_tx_fails_wrong_tx_version() {
 #[test]
 fn given_contract_declare_on_openzeppelin_account_then_it_works() {
     new_test_ext().execute_with(|| {
+        SequencerAddress::<MockRuntime>::put(DEFAULT_SEQUENCER_ADDRESS);
```

Comment:
Could you provide a utility method wich do all of the basic setup things we do in each test?

```
SequencerAddress::<MockRuntime>::put(DEFAULT_SEQUENCER_ADDRESS);
System::set_block_number(0);
run_to_block(2);
``` 

And use it everywhere.

It will help future contributors

Reply:
Name it, basic_test_setup, or something along the lines